custom/icds/static/icds/js/manage_hosted_ccz.js

```diff
@@ -21,6 +21,10 @@ hqDefine('icds/js/manage_hosted_ccz', [
             self.fileName = fileName;
             self.note = note;
             self.status = status;
+            self.rowColor = function () {
```

Comment:
my original attempt was to use `class` but i just could not get it working with knockout js so i then fell back to `color`.
I had rowClass as a computed property which returned `info`, 'danger` etc and then i did `data-bind=class: rowClass` on `tr` but the class never came into action.
I will be happy to revert this change to class.

Reply:
`class` was added in 3.5 and we're only on 3.3: https://github.com/knockout/knockout/releases

Could you use `css`? https://knockoutjs.com/documentation/css-binding.html

I also wouldn't object to upgrading to 3.5.

Reply:
Updated to use `css`. 
Upgrade would need a robust testing.